lms/djangoapps/verify_student/views.py

```diff
@@ -856,6 +858,67 @@ def submit_photos_for_verification(request):
     return HttpResponse(200)
 
 
+def _compose_message_reverification_email(course_key, user_id, relates_assessment, photo_verification, status, is_secure):
```

Comment:
Change the name of method to `_compose_reverification_email` and add docstring:
`""" Create Re-verification email content. """`